src/util/Constants.js

```diff
@@ -19,7 +19,7 @@ const AllowedImageFormats = ['webp', 'png', 'jpg', 'jpeg', 'gif'];
 const AllowedImageSizes = Array.from({ length: 9 }, (e, i) => 2 ** (i + 4));
 
 function makeImageUrl(root, { format = 'webp', size } = {}) {
-  if (typeof size !== 'number') throw new TypeError('INVALID_TYPE', 'size', 'number');
+  if (size && typeof size !== 'number') throw new TypeError('INVALID_TYPE', 'size', 'number');
```

Comment:
```suggestion
  if (typeof size !== 'undefined' && typeof size !== 'number') throw new TypeError('INVALID_TYPE', 'size', 'number');
```

Reply:
Does this work too? Looks cleaner imo
```suggestion
  if (!['undefined', 'number'].includes(typeof size)) throw new TypeError('INVALID_TYPE', 'size', 'number');
```

Reply:
Sure, that works too